components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -11,6 +11,8 @@ module lin_strat_chem
 !       --added prescribed O3LBS in netcdf file (prescribed CMIP6 historical surface ozone below 925 mb). And Linoz surface ozone are relaxed to this profile in 2-days within the last 9 surface layers
 !      added 30-day e-fold decay subroutine (lin_strat_efold_decay) for ChemUCI tropospheric species assigned to the namelist, fstrat_efold_list   
 !     20 Sep 2021 -- Qi Tang (LLNL), -- added 3D tropopause
+!     26 Oct 2021 -- Hsiang-He Lee (LLNL) -- run both O3 and O3linoz in subrountine 
```

Comment:
You probably want to revise this to reflect what the final changes are.

Reply:
Changed.